src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import { Checkbox, Form, FormGroup, Select, SelectOption, Toolbar, ToolbarGroup, ToolbarItem } from '@patternfly/react-core';
+import { DownloadIcon } from '@patternfly/react-icons';
+import routerParams from '../../../../Utilities/RouterParams';
+import { SimpleTableFilter } from '../../../../PresentationalComponents/SimpleTableFilter';
+import debounce from 'lodash/debounce';
+import propTypes from 'prop-types';
+
+const CVSSOptions = [
+    { value: 'all', label: 'All', disabled: false, from: '', to: '' },
+    { value: '0to3', label: 'Lower than 3', disabled: false, from: '', to: 3 },
+    { value: '3to7', label: '3 - 7', disabled: false, from: 3, to: 7 },
+    { value: '7to10', label: '7 - 10', disabled: false, from: 7, to: 10 }
+];
+
+class VulnerabilitiesCveTableToolbar extends Component {
+    state = {
+        defaultConfig: {
+            filter: '',
+            // eslint-disable-next-line camelcase
+            cvss_score: 'all',
+            // eslint-disable-next-line camelcase
+            show_all: 'False'
+        },
+        toolbarConfig: {}
+    }
+    changeFilterValue = debounce(value => this.setState(
+        {
+            toolbarConfig: { ...this.state.toolbarConfig, filter: value }
+        },
+        this.apply
+    ), 400);
+
+    apply = () => this.props.apply(this.state.toolbarConfig);
+
+    changeCVSSValue = (value, options) => {
+        const target = options.find(item => item.value === value);
+        // eslint-disable-next-line camelcase
+        this.setState({ toolbarConfig: { ...this.state.toolbarConfig, cvss_from: target.from, cvss_to: target.to }}, this.apply);
+    }
+
+    changeCheckboxValue = (value) => {
+        // eslint-disable-next-line camelcase
+        this.setState({ toolbarConfig: { ...this.state.toolbarConfig, show_all: !value }}, this.apply);
+    }
+
+    getCVSSValue = (options) => {
+        const option = options.find(
+            // eslint-disable-next-line camelcase
+            item => item.from === this.state.toolbarConfig.cvss_from && item.to === this.state.toolbarConfig.cvss_to
+        );
+        return option ? option.value : options[0].value;
+    }
+
+    render() {
+        const { showAllCheckbox, downloadReport, totalNumber } = this.props;
+
+        return (
+            <React.Fragment>
```

Comment:
friendly reminder of `<>` or `<Fragment>` shorthand https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html#keyed-fragments

Reply:
I had some problems making `<>` to run and if consumer is using older version of react than 16.2 he's gonna bump into problem with undefined component. Gonna need it there for now, but this can be removed later.